<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ingresoBienes</class>
 <widget class="QWidget" name="ingresoBienes">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>625</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>975</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ingreso de Bienes al Inventario de Activo Fijo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ContablesPrincipal.qrc">
    <normaloff>:/images/ico_alpha_RegistryDefrag_32x32.png</normaloff>:/images/ico_alpha_RegistryDefrag_32x32.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>941</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Datos Generales: </string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Unidad: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombreUnidad">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Departamento:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombreDpto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Tipo de Adquisicion:</string>
    </property>
    <widget class="QRadioButton" name="fondoGeneral">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Fondo General</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fondosPropios">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Fondos Propios</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="donado">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>84</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Donado</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Proveedor: </string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Factura: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="noFactura">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha Factura:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="fechaFactura">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2009</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CEF. No.:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CEF">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
    <widget class="QComboBox" name="nombreProveedor">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Acuerdo C.S.U: </string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>No.:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="numeroAcuerdoCSU">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>15</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="fechaAcuerdo">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2010</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>941</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Bien:</string>
   </property>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Especificacion Bien:</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Cuenta: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>46</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Clase:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="numeroCuenta">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nombreClase">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="cuentaAsignada">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Especifico: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nombreEspecifico">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Marca:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Modelo:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="marca">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="modelo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Serie:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serie">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha Adquisicion:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="fechaAdquisicion">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>2010</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valorBien">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Spanish" country="ElSalvador"/>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sector Destino:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sectorDestino">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="descripcion">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descripcion:</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="botonAgregarBien">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>320</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Agregar Bien</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../ContablesPrincipal.qrc">
     <normaloff>:/iconos/mas.png</normaloff>:/iconos/mas.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="botonEliminarBien">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>360</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Eliminar Bien</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../ContablesPrincipal.qrc">
     <normaloff>:/iconos/menos.png</normaloff>:/iconos/menos.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="finalizarTransaccion">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>580</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Finalizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>580</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QToolButton" name="nuevoBien">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>400</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Nuevo Bien</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../ContablesPrincipal.qrc">
     <normaloff>:/iconos/filenew.png</normaloff>:/iconos/filenew.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="observaciones">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>891</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>148</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalIngreso">
   <property name="geometry">
    <rect>
     <x>772</x>
     <y>520</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>520</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Total:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Observaciones:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablaBienes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>901</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nombreUnidad</tabstop>
  <tabstop>fondoGeneral</tabstop>
  <tabstop>fondosPropios</tabstop>
  <tabstop>donado</tabstop>
  <tabstop>noFactura</tabstop>
  <tabstop>CEF</tabstop>
  <tabstop>fechaFactura</tabstop>
  <tabstop>numeroAcuerdoCSU</tabstop>
  <tabstop>fechaAcuerdo</tabstop>
  <tabstop>numeroCuenta</tabstop>
  <tabstop>nombreClase</tabstop>
  <tabstop>marca</tabstop>
  <tabstop>modelo</tabstop>
  <tabstop>serie</tabstop>
  <tabstop>descripcion</tabstop>
  <tabstop>fechaAdquisicion</tabstop>
  <tabstop>valorBien</tabstop>
  <tabstop>sectorDestino</tabstop>
  <tabstop>botonAgregarBien</tabstop>
  <tabstop>botonEliminarBien</tabstop>
  <tabstop>nuevoBien</tabstop>
  <tabstop>observaciones</tabstop>
  <tabstop>finalizarTransaccion</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>totalIngreso</tabstop>
  <tabstop>tablaBienes</tabstop>
  <tabstop>nombreEspecifico</tabstop>
  <tabstop>cuentaAsignada</tabstop>
  <tabstop>nombreDpto</tabstop>
 </tabstops>
 <resources>
  <include location="../ContablesPrincipal.qrc"/>
 </resources>
 <connections/>
</ui>
